doc/source/user_guide/timeseries.rst

```diff
@@ -1780,6 +1777,48 @@ natural and functions similarly to :py:func:`itertools.groupby`:
 
 See :ref:`groupby.iterating-label` or :class:`Resampler.__iter__` for more.
 
+.. _timeseries.adjust-the-start-of-the-bins:
+
+Use `origin` or `offset` to adjust the start of the bins
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The bins of the grouping are adjusted based on the beginning of the day of the time series starting point. This works well with frequencies that are multiples of a day (like `30D`) or that divides a day (like `90s` or `1min`). But it can create inconsistencies with some frequencies that do not meet this criteria. To change this behavior you can specify a fixed timestamp with the argument ``origin``.
+
+For example:
+
+.. ipython:: python
+
+    start, end = "1/10/2000 02:00:00", "1/20/2000 02:00"
+    middle = "1/15/2000 02:00"
+    rng = pd.date_range(start, end, freq="1231min")
```

Comment:
Could we use a more digestible frequency example? e.g. `7 min` or something (and with the resampling frequency as well)

Reply:
Yeah, you are right, I did the change using 7min and 17min everywhere :)